Content.Shared/Cognislime/CognislimeComponent.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Shared.Whitelist;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Cognislime;
+
+/// <summary>
+/// Makes objects the entity is applied to sentient and a ghost role.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(true)]
```

Comment:
dont need AGCS and you dont use the event so true isnt needed regardless
replace it with Access(typeof(SharedCognislimeSystem))

Reply:
Done. Thanks once again for the review, I'd be totally lost without it.